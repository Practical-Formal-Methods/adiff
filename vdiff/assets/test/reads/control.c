int main() {
	int x;

	while (x < 5) {; }

	for (int i = x; i < 5; i++) {; }

	if (x > 5) {; }

	switch (x) {
		case 0: return 1;
		default: return 0;
	}

}
