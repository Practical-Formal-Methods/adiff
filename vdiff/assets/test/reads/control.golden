int main()
{
    int x;
    __VERIFIER_read(x);
    while (x < 5)
    {
        ;
    }
    __VERIFIER_read(x);
    for (int i = x; i < 5; i++)
    {
        ;
    }
    __VERIFIER_read(x);
    if (x > 5)
    {
        ;
    }
    __VERIFIER_read(x);
    switch (x)
    {
    case 0:
        return 1;
    default:
        return 0;
    }
}